### DISCUSSION TITLE
Add pinch-to-zoom

### DISCUSSION THREAD

**Discussion by dronnikovigor** [Radiokot/photoprism-android-client#117] [General]
When i open photo and try to zoom with 2-finger-gestue it just moves the photo. The only way is to zoom with double tap. And then you can use pinch-to-zoom. Sometimes it's not so convinient.

**Comment by Radiokot** [comment]
Sounds like a bug, but I can't reproduce it. On my devices, pinch gesture works fine. Will keep it in mind.

**Reply by dronnikovigor** [reply]
Checked one more time. Seems like it works, but in very small area and you should be very accurate to move fingers in diagonals (towards to edges of photo).
But if you open a new photo and try to move in up & down directions with 2 fingers, photo will jump. 

I'll try to record sample video to explain better :)

**Reply by dronnikovigor** [reply]
Here is the video:
https://github.com/Radiokot/photoprism-android-client/assets/4175117/25254f3b-34ad-4036-a31a-380cd18928a3

**Reply by Radiokot** [reply]
OMG, what a mess 🤯 Have never experienced such behavior, though. Can you please give more info about your device? Android version, manufacturer, and if this "go back" gesture you use is a system feature or a 3rd party app.

**Reply by dronnikovigor** [reply]
Pixel 6, Android 14. Go back gesture is system default. I tried to switch to 3 button navigation - same issue.
For me it looks like prev/next & close gesture has higher priopity than zoom. Maybe i'm wrong. I tried same behavior in Google Photos, and it doesn't do anything at all with photo(close or show next/prev) if you use more than one finger .

**Reply by ippocratis** [reply]
If you double tap the image first then the pinch sensitivity  works as intended otherwise yeah its kinda buggy

**Comment by Radiokot** [comment]
Created an issue for this
https://github.com/Radiokot/photoprism-android-client/issues/118
